microsoft-edge/webdriver-chromium/index.md

```diff
@@ -351,6 +351,24 @@ docker run -d -p 9515:9515 mcr.microsoft.com/msedge/msedgedriver
 
 For more information, navigate to the [msedgedriver container on Docker Hub][DockerHubMsedgedriver].  
 
+## Application Guard
+
+<!-- was "Windows Defender Application Guard" renamed to "Microsoft Defender Application Guard"; are those the same thing? -->
+
+Microsoft Edge (Chromium) windows that use Windows Defender Application Guard (WDAG) cannot be automated or manipulated using WebDriver-related or Selenium-related technologies.
```

Comment:
```suggestion
Untrusted sites that use Microsoft Defender Application Guard (Application Guard) cannot be automated or manipulated with Microsoft Edge Driver. Application Guard launches untrusted sites in a container and the container does not expose the remote debugging port Microsoft Edge Driver needs to communicate with the site. Your enterprise administrator defines what are trusted sites, cloud resources, and internal networks. Everything not in the trusted sites list is considered untrusted. InPrivate windows and sites on the trusted sites list can still be automated with Microsoft Edge Driver. For more information about Application Guard, navigate to: 
```

Reply:
Let's just use the "Microsoft Defender Application Guard" name from: https://docs.microsoft.com/en-us/deployedge/microsoft-edge-security-windows-defender-application-guard and I like just referring to it as Application Guard from then on.

Reply:
+1

Reply:
Note: URL says "windows-defender", top-of-page title says "Microsoft Defender".